crates/toolchain/build/src/lib.rs

```diff
@@ -349,24 +387,17 @@ pub fn build_guest_package(
         .expect("cargo build failed");
     let stderr = child.stderr.take().unwrap();
 
-    tty_println(&format!("{}: Starting build for {RUSTC_TARGET}", pkg.name));
+    tty_println(&format!("openvm build: Starting build for {RUSTC_TARGET}"));
 
     for line in BufReader::new(stderr).lines() {
-        tty_println(&format!("{}: {}", pkg.name, line.unwrap()));
+        tty_println(&format!("openvm build: {}", line.unwrap()));
     }
 
     let res = child.wait().expect("Guest 'cargo build' failed");
     if !res.success() {
         Err(res.code())
     } else {
-        Ok(get_dir_with_profile(
-            &target_dir,
-            profile,
-            target_filter
-                .as_ref()
-                .map(|t| t.kind == "example")
-                .unwrap_or(false),
-        ))
+        Ok(get_dir_with_profile(target_dir, profile, false))
```

Comment:
maybe we don't need the last arg of this function anymore, but ok to leave as well